plug-ins/prometheus/architecture.md

```diff
@@ -0,0 +1,107 @@
+# Prometheus Plugin Architecture
+
+This document describes the architecture for the Prometheus Plugin (eosio::prometheus_plugin).  The Prometheus Plugin instruments nodes in order to allow
+data to be collected from [Prometheus](https://prometheus.io/).  The requirements for this plugin are described in this github issue:
```

Comment:
Now that there's no link, "this github issue" is a little out of place.  "extracted from this github issue:" and restore the link?  Preserves traceability while improving resiliency.